src/tests/stress/pedsim/Ped_util/Regions.enc

```diff
@@ -332,19 +323,22 @@ active class Box
         case unused  => true
      end
   end
-  def move() : bool
+  def private yolo_item(a:Maybe[Item]) : Item
```

Comment:
the name yolo is taken from Tobias that suggested to add a method with that name for doing exactly what I am doing in these methods ('yolo_int', 'yolo_string', 'yolo_item'). The reason 'yolo_item' is a private method is that 'Item' is a local class.